src/main/java/seedu/address/logic/commands/ChargeCommand.java

```diff
@@ -0,0 +1,104 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Collection;
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.attendance.AttendanceEntry;
+import seedu.address.model.charge.Charge;
+import seedu.address.model.charge.ChargeDate;
+import seedu.address.model.pet.Pet;
+
+
+/**
+ * Computes a month's charge of a pet identified using it's displayed index from the address book.
+ */
+public class ChargeCommand extends Command {
+
+    public static final String COMMAND_WORD = "charge";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Computes a month's charge of the pet identified "
+            + "by the index number used in the last pet listing.\n"
+            + "Parameters: INDEX (must be a positive integer)"
+            + "m/[MM-yyyy] c/[cost]\n"
+            + "Example: " + COMMAND_WORD + " 1 "
+            + "m/03-2022 c/200";
+
+    public static final String MESSAGE_COMPUTE_CHARGE_SUCCESS = "Computed charge of Pet: $%.2f";
```

Comment:
Maybe can consider rephrasing the message to something like `__[NAME]__ should be charged _____ for the month of _____` if you are able to put in the name and a specific month into the `generateSuccessMessage` method. Else I think this should work too!

Reply:
Good suggestion by @tsumian.`Pet#getName()` can assist with [NAME].

Reply:
Good idea @tsumian, that would make WoofAreYou more user-centric. Thanks for the pointer too @Dineshraj555!